cabot/cabotapp/models.py

```diff
@@ -580,6 +581,9 @@ def should_run(self):
 
             # The last thing to verify is that the check hasn't run within the frequency
 
+        if self.run_window and not self.run_window.active(timezone.now()):
+            return False
```

Comment:
Sorta-open question - what should we do if the check is failing when the run window expires? The check will be in the failed state until the next run window, and will continuously alert.

We could keep running the check, or maybe just not alert outside (run window +/- frequency).

Reply:
Yeah let's also not alert based on the check status outside the run window? It should run immediately after the run window starts so I don't think we need to add the frequency to that...but maybe we could add a couple minutes since the check could run slightly after the first `update_service`?